problems/isMultipleOfFive.js

```diff
@@ -13,8 +13,12 @@
 * returns: false (because the string "ten" is not a number)
 */
 
-function isMultipleOfFive() {
-
-}
+function isMultipleOfFive(n) {
+    if (n % 5 === 0){
```

Comment:
Nice job. When returning true or false based on if the condition evaluates to true or false, just return the condition.